src/applications/personalization/rated-disabilities/components/TotalRatedDisabilities.jsx

```diff
@@ -0,0 +1,132 @@
+    const message = (
+      <span>
+        <p>We don't have a disability rating for you in our system.</p>
+        <h4>Want to add disabilities?</h4>
+        <p>
+          If you believe you have disabilities related to your military service,
+          you can submit a claim to update your disability rating.
+        </p>
+        <a href="#" className="usa-link">
+          Submit a new claim
+        </a>
+      </span>
+    );
+    content = (
+      <AlertBox
+        headline="No disability rating to show"
+        content={message}
+        status="info"
+        isVisible
+      />
+    );
+  } else {
+    content = (
+      <span>
+        <div className="total-rated-disabilities">
+          <div className="vads-l-row">
+            <div className="vads-l-col--12">
+              <h1>Your disability rating</h1>
+            </div>
+          </div>
+          <div className="vads-l-row">
+            <div className="vads-l-col--12 medium-screen:vads-l-col--6 small-desktop-screen:vads-l-col--9 vads-u-padding-right--1p5">
+              <p>
+                <strong>
+                  Your combined disability rating is {totalDisabilityRating}%
+                </strong>
+                . This rating determines the amount of compensation you'll get.
+                The {''}
+                <a href="#"> individual disability ratings </a>
+                {''}
+                below are what we used to calculate your combines disability
+                rating.
+              </p>
+              <p>
+                This rating does not include any pending disabilities you may
+                have applied for. You can check the status of your disability
+                claims or appeals with the Claim Status Tool.You can also view
+                your uploaded documents or request help with a claim or appeal.
+              </p>
+              <p>
+                <a href="https://www.youtube.com/watch?v=oM7oYzL2DCg">
+                  Check your claims or appeals
+                </a>
+              </p>
+            </div>
+            <div className="vads-l-col--12 medium-screen:vads-l-col--6 small-desktop-screen:vads-l-col--3">
+              <PercetnageCalloutBox
+                value={totalDisabilityRating}
+                isPercentage
+                label="Your combined VA disability rating"
+              />
+            </div>
+          </div>
+          <div className="vads-l-row">
+            <div className="vads-l-col--12 medium-screen:vads-l-col--6 small-desktop-screen:vads-l-col--8">
+              <h3>What if I disagree with my rating?</h3>
+              <p>
+                If you disagree with your rating, you can file an appeal. You'll
+                need to do this within 1 year of getting your decision notice.
+              </p>
+              <p>
+                <a href="#">Learn how to file an appeal</a>
+              </p>
+            </div>
+          </div>
```

Comment:
I don't think this section is necessary anymore. 
![Screen Shot 2019-10-11 at 11 40 06 AM](https://user-images.githubusercontent.com/15097156/66676263-e6163d00-ec1b-11e9-8a45-a4ae294d1b68.png)

Reply:
Would it be possible to align the text of the left column with the top of the callout box? I think this can be done by just adjusting the margins of that top `<p>` tag. 

<img width="678" alt="Screen Shot 2019-10-11 at 11 50 45 AM" src="https://user-images.githubusercontent.com/15097156/66676956-8456d280-ec1d-11e9-9169-1bf58e95a91e.png">

Reply:
Removing the unneeded content block and adjusted padding in the column to align content in commit 0e211ef

Reply:
<img width="649" alt="Screen Shot 2019-10-11 at 12 38 20 PM" src="https://user-images.githubusercontent.com/1899695/66679840-0b0eae00-ec24-11e9-881d-e3a2b59c58cc.png">